src/berryimu.py

```diff
@@ -0,0 +1,243 @@
+from typing import ClassVar, Mapping, Tuple
+from typing import ClassVar, Mapping, Any, Dict, Optional
+from typing_extensions import Self
+
+from viam.proto.app.robot import ComponentConfig
+from viam.proto.common import ResourceName
+from viam.resource.base import ResourceBase
+from viam.resource.types import Model, ModelFamily
+
+from viam.components.movement_sensor import MovementSensor
+from viam.logging import getLogger
+from viam.proto.common import GeoPoint, Orientation, Vector3
+from smbus import SMBus
+from . import constants
+from . import utils
+import time
+import math
+import threading
+import asyncio
+from .orientation_vector import euler_angles_to_orientation_vector
+
+
+LOGGER = getLogger(__name__)
+dt = 0.08
+
+class Berryimu(MovementSensor):
+    MODEL: ClassVar[Model] = Model(ModelFamily("viam", "movement_sensor"), "berryimu")
```

Comment:
your model namespace needs to match the namespace that you're going to upload it in.

Reply:
Oh I see so make it viam-labs? How do I know if I have access to a namespace?

Reply:
you can try uploading anythign to a namespace through cli, but also - you have access to any orgs that you are an owner of in app.viam.com, their public namespaces will be available to you onc you log into the cli. 

I'm also okay if you want to upload it under your own namespace.